### PR TITLE
Add link to discussion on implementing an outbox

### PR DESCRIPTION
**Pull request by kbaley**
The [previous PR](https://github.com/MicrosoftDocs/architecture-center/pull/3490) was closed but I'd like to have it re-considered. The webinar is technology agnostic and is a discussion on the concepts and patterns described in the article, similar to the link right above it.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@EdPrice-MSFT 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by kbaley** [comment]
@Jak-MS @EdPrice-MSFT Let me know if you need anything from my end. I tried to reword it to make it clearer this was a discussion on how to implement your own outbox but I'm open to suggestions.

**Comment by adamralph** [comment]
@Jak-MS @EdPrice-MSFT I believe this would be a valuable addition to this document. As @kbaley said:

> The webinar is technology agnostic and is a discussion on the concepts and patterns described in the article, similar to the link right above it.

It seems to be a useful resource for people who want to investigate the outbox pattern more deeply.

**Comment by EdPrice-MSFT** [comment]
The topic is good. But this is specific to NServiceBus, a Particular product:

https://particular.net/nservicebus
